go-controller/pkg/ovn/pods.go

```diff
@@ -182,7 +182,8 @@ func (oc *Controller) addLogicalPort(pod *kapi.Pod) {
 
 		out, stderr, err = util.RunOVNNbctlUnix("--may-exist", "lsp-add",
 			logicalSwitch, portName, "--", "lsp-set-addresses", portName,
-			fmt.Sprintf("%s %s", macAddress, ipAddress))
+			fmt.Sprintf("%s %s", macAddress, ipAddress), "--", "--if-exists",
+			"clear", "logical_switch_port", portName, "dynamic_addresses")
```

Comment:
If pod already has annotation of ip/mac addresses, why do we need to use dynamic addresses? I am trying to understand under which circumstance, it can have duplicate IP. Also, if we use dynamic addresses here, we don't need L180 & L181 anymore.